app/views/partials/multi_factor_authentication/_mfa_selection.html.erb

```diff
@@ -30,6 +30,13 @@
     <% end %>
     <span id="two_factor_options_form_selection_info_<%= option.type %>" aria-hidden="true" class="usa-checkbox__label-description">
       <%= option.info %>
+      <% if option.recommended? %>
+        <br />
+        <%= render TagComponent.new(
+              informative: true,
+              class: 'margin-top-1',
+            ).with_content(t('two_factor_authentication.recommended')) %>
+      <% end %>
```

Comment:
The 'Added' tag is similar to this but has the `aria-hidden="true"` attribute. I'm wondering if it should be on the Recommended tag also, or should it not be on the Added tag either.